cddl/mask-type.cddl

```diff
@@ -1,4 +1,3 @@
-mask-operators /= op.eq
 mask-type = bstr
-mask-expression = [ mask-operators, value: mask-type, mask: mask-type]
-tagged-mask-expression = #6.60040( mask-expression )
\ No newline at end of file
+$masked-value-type /= mask-type
+$masked-value-type /= tagged-masked-raw-value
```

Comment:
```suggestion
$masked-value-type /= tagged-masked-raw-value
$masked-value-type /= tagged-bytes
```

Reply:
Suggest we do this to allow cases when there is no mask!

Reply:
As in the new one Mask is mandatory!

Reply:
> As in the new one Mask is mandatory!

Not sure what you mean by mandatory. `$xxx-type` type-choice variants are optional.